libs/services/src/SlippageService/SlippageService.ts

```diff
@@ -1,20 +1,53 @@
+import { UsdRepository, usdRepositorySymbol } from '@cowprotocol/repositories';
+import { injectable, inject } from 'inversify';
+
 /**
  * BPS (Basis Points)
  */
 export type Bps = number;
 
 export interface SlippageService {
-  getSlippageBps(quoteTokenAddress: string, baseTokenAddress: string): Bps;
+  getSlippageBps(
+    quoteTokenAddress: string,
+    baseTokenAddress: string
+  ): Promise<Bps>;
 }
 
-// TODO: Find good name for the implementation, as Leandro don't like "Impl" suffix, just don't want to couple it to add Coingecko in its name (as that would couple it to the data-source, and the adding a name to specify the algorithm might complicate the name, as this will use some custom logic based on standard deviation, so for now as we plan to have just one implementation, I want to keep it simple. But happy to get ideas here)
+export const slippageServiceSymbol = Symbol.for('SlippageService');
```

Comment:
Why the symbol? Is that a common pattern with `injectable`?
Also, since it's a const, should it be all CAPS?

Reply:
Yep, it was required. Followed their getting started guide. I think its similar to Spring, when you give a name to a bean. Here they use symbols so its not loose strings, and this way they can check for collisions

<img width="1123" alt="image" src="https://github.com/cowprotocol/bff/assets/2352112/e023c9a2-61d6-4b5a-bbe0-a067d2c94e2c">